modules/distribution/src/main/java/org/openlmis/distribution/domain/ChildCoverageLineItem.java

```diff
@@ -40,6 +40,14 @@
   private Integer outreach11Months;
   private Integer healthCenter23Months;
```

Comment:
In cases like this (in the backend-Java as well as frontend JS), the meanings of "outreach11Months" and "healthCenter23Months" have changed. They now may be better described respectively as "totalOutreach11Months" and "totalHealthCenter23Months."